development/README.md

```diff
@@ -0,0 +1,17 @@
+# MAP - Development Environment
+
+## Description
+This directory contains all that is required to spin-up a local development environment for the MAP project.
+
+## Requirements
+- Docker
+- `docker-compose`
+
+## Usage
+### Spin up development environment
+`sh start.sh`
```

Comment:
This is here so that the developer does not have to type `docker-compose up/down` each time they want to spin down/up the dev env. Also, this allows for more customization to be added when spinning up locally.